pkg/roachprod/vm/gce/utils.go

```diff
@@ -31,7 +31,11 @@ import (
 const gceDiskStartupScriptTemplate = `#!/usr/bin/env bash
 # Script for setting up a GCE machine for roachprod use.
 
-set -x
+# ensure any failure fails the entire script
+set -eux
+
```

Comment:
Can we add a message in the beginning "Sending output to /var/log/roachprod_startup.log"?

Reply:
Done in another way